modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/customer/fragments/side-menu/index.html

```diff
@@ -4,22 +4,22 @@
 <div class="customer-portal-side-menu">
 	<ul class="list-unstyled">
 		<li>
-			<button class="active btn btn-menu btn-sm customer-portal-side-menu-button skeleton" id="custumer-portal-overview" type="button" value="overview">
+		<button class="active btn btn-menu btn-sm customer-portal-side-menu-button skeleton" id="custumer-portal-overview" type="button" value="overview">
 				Overview
 			</button>
 		</li>
 		<li>
-			<button class="align-items-center btn btn-menu btn-sm justify-content-between skeleton text-neutral-10" id="customer-portal-toggle-products" type="button">
+		<button class="align-items-center btn btn-menu btn-sm justify-content-between skeleton text-neutral-10" id="customer-portal-toggle-products" type="button">
 				Product Activation
-				<svg class="left lexicon-icon lexicon-icon-angle-right-small" focusable="false" id="customer-portal-arrow" role="presentation">
+			<svg class="left lexicon-icon lexicon-icon-angle-right-small" focusable="false" id="customer-portal-arrow" role="presentation">
 					<use xlink:href="${clayIconSVG}#angle-right-small" />
 				</svg>
 			</button>
 
-			<ul class="list-unstyled ml-3 overflow-hidden" id="customer-portal-products" />
+		<ul class="list-unstyled ml-3 overflow-hidden" id="customer-portal-products" />
 		</li>
 		<li>
-			<button class="btn btn-menu btn-sm customer-portal-side-menu-button skeleton" id="custumer-portal-members" type="button" value="team">
+		<button class="btn btn-menu btn-sm customer-portal-side-menu-button skeleton" id="custumer-portal-members" type="button" value="team_members">
```

Comment:
Hey @claricebrasil, we do not use side menu as a fragment anymore, please modify the SideMenu React component